omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml

```diff
@@ -0,0 +1,1378 @@
+                                            <div class="col">
+                                            <span id="checkboxRadio">
+                                                <span class="ARTYesNoEnableDisable1">
+                                                    <label>Family Planning? </label>
+                                                    <obs conceptId="90238" answerConceptIds="90082,90081" answerLabels="Yes,No" />
+                                                </span>
+                                            </span>
+                                            </div>
+                                        </div>
+                                        <h4>Family Planning Method:</h4>
+                                        <div class="row">
+                                            
+                                            <div class="col">                           
+                                            <span class="ARTYesNoEnableDisableChild1">
+                                               
+                                                <obs conceptId="374" answerConceptId="190" answerLabel="1 - Condoms"/>
+                                                <excludeIf velocityTest="$patient.gender =='M' ">
+                                                    <span class="lablex2">
+                                                        <obs conceptId="374" answerConceptId="780"
+                                                             answerLabel="2 - Oral Contraceptive pills"/>
+                                                        <obs conceptId="374" answerConceptId="5279"
+                                                             answerLabel="3 - Injectable/Implantable hormonies"/>
+                                                        <obs conceptId="374" answerConceptId="5278"
+                                                             answerLabel="4 - Diaphram/Cervical Cap"/>
+                                                        <obs conceptId="374" answerConceptId="5275"
+                                                             answerLabel="5 - Intra uterine device (IUD)"/>
+                                                    </span>
+                                                </excludeIf>
+                                                <obs conceptId="374" answerConceptId="99265"
+                                                     answerLabel="6 - Vasectomy/Tubal ligation"/>
+                                            </span>
+                                            </div>
+                                        </div>
+                                    </li>
+                                </excludeIf>
+                               
+                            </ul>
+                        
+                    </div>
+
+                    <div class="tab-pane container p-5 fade" id="diagnosis">
```

Comment:
@semujju why the classes container and p-5?

Reply:
The containers adds padding on the left and right, then the p-5 allows for top and bottom padding.